_posts/2024-10-10-Vorstellung-Agnes-Seipelt.md

```diff
@@ -0,0 +1,28 @@
+
+
+Noch in meiner Zeit im Bruckner Projekt wurde ich zu einem Arbeitstreffen vom Projekt Beethovens Werkstatt eingeladen, das das Konzept der Harmonischen Analyse im Rahmen des 2 Moduls aufgreifen wollte. Nach Ende des Bruckner-Projekts konnte ich dann glücklicherweise direkt bei Beethovens Werkstatt anfangen und habe die Harmonische Analyse (mit einigen Anpassungen und Verbesserungen) mit Hilfe von Johannes Kepper als eine der Perspektiven in der Vide-App von [Modul 2](https://beethovens-werkstatt.de/modul-2/) implementiert. 
+
+Seit 2019 bin ich nun Teil des Detmolder Teams von Beethovens Werkstatt. Das Projekt erforscht kompositorische Denk- und Arbeitsprozesse am Beispiel von Beethoven, wobei die Methoden der genetischen Textkritik weiterentwickelt und gleichzeitig digitale Formen zur Präsentation der Forschungsergebnisse geschaffen werden. Die ursprüngliche, konzeptuelle Aufteilung der Arbeitsstellen in „philologisch“ (Arbeitsstelle Bonn) und „digital/technisch“ (Arbeitsstelle Detmold) ist seit Projektbeginn auf natürliche Weise immer mehr aufgeweicht worden. So werden die, natürlich je nach Modul immer etwas anders gelagerten, Aufgaben wie das Nachdenken über und die Konzeption unserer Datenmodellen, das Anfertigen von Musikcodierungen mit MEI, philologische Arbeiten wie das Transkribieren von Skizzen und anderen Handschriften, das Konzipieren von Transkriptions- und Codierungsrichtlinien, das Transformieren von Codierungen sowie Korrekturlesearbeiten und das Verfassen von begleitenden Texten von allen MitarbeiterInnen meist gleichermaßen übernommen. 
+
+Daneben beschäftige ich mich im Rahmen meiner Dissertation mit der Bagatelle Nr. 6 aus Op. 126 von Beethoven. Anknüpfend an Konzepte aus der „Werkstatt“ möchte ich eine genetische Edition der Bagatelle erstellen, die Konzepte der Werkstatt nachnutzt, neue Ansätze diskutiert und unterschiedliche Perspektiven auf die Quellen und den Text ermöglicht. Dazu sollen alle Quellen (Skizzen, zwei Werkniederschriften und die Originalausgabe) einschließlich aller Variantenstellen transkribiert und textgenetisch untersucht werden, sodass interne und dokumentübergreifende Text- und Schreibchronologien formuliert werden können, die in einer Darstellung mit den SVG-Shapes auf dem Faksimile verknüpft werden. Darüber hinaus sollen Konzepte für die Verknüpfung bzw. die Beschreibung der Beziehungen der Skizzen und Notate untereinander und zum Werktext modelliert werden. 
```

Comment:
In diesem (letzten) Absatz müsstest du dich entscheiden, ob du Werkstatt mit oder ohne Anführungszeichen schreibst.